addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -83,12 +83,14 @@ $o-wslides-fs-side-width: 300px;
         }
     }
 
-    .o_wsildes_quiz_question_input {
-        .o_wslides_js_quiz_is_correct > label,
-        .o_wslides_js_quiz_add_answer,
-        .o_wslides_js_quiz_remove_answer {
+    .o_wslides_js_lesson_quiz {
+        i.o_wslides_js_quiz_icon {
             cursor: pointer;
         }
+
+        i.o_wslides_js_quiz_icon:hover {
+            color: black !important;
+        }
```

Comment:
I would remove this change.
For multiple reasons:
- It's not related to mobile ("hover" problem)
- It's not really a "fix" (works fine, just not very bioutiful)
- (bonus: I don't like "important")